soh/soh/Enhancements/bootcommands.c

```diff
@@ -65,7 +65,7 @@ void BootCommands_ParseBootArgs(s32 argc, char** argv)
     for (i = 0; i < argc; i++)
         DebugArena_Free(argv[i]);
 
-    DebugArena_Free(argv);
+    //DebugArena_Free(argv);
 }
```

Comment:
Should we not just remove this commented out code? Git will help us restore this if needed -- same for other instances of this